intake/cli/server/server.py

```diff
@@ -353,6 +353,8 @@ def post(self):
             msg = dict(format=chunk_encoder.format_name,
                        compression=chunk_encoder.compressor_name,
                        container=source.container, data=data)
+            # Is this double serializing / copying?
+            # We're passing a Buffer object here.
```

Comment:
I imagine msgpack doesn't do anything to the bytestring here (except perhaps copy).
It would be a larger discussion, perhaps worth having, whether this framing is useful at all. It would be far more efficient to do `context.serialize(obj).write_to(self)` here - both sides have already agreed that this is a piece of dataframe.